sklearn/linear_model/logistic.py

```diff
@@ -623,20 +623,20 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
         mask = (y == pos_class)
         y_bin = np.ones(y.shape, dtype=np.float64)
         y_bin[~mask] = -1.
+        # for compute_class_weight
+
+        if class_weight in ("auto", "balanced"):
```

Comment:
Comment with the version is most helpful. ``#TODO`` is fine. I grep for the version so ``0.19`` in this case I guess